components/ai_chat/browser/ai_chat_tab_helper.cc

```diff
@@ -407,8 +407,8 @@ void AIChatTabHelper::MakeAPIRequestWithConversationHistoryUpdate(
   // don't have to do this weird substitution.
   std::string question_part;
   if (turn.text == "Summarize this video") {
-    question_part =
-        l10n_util::GetStringUTF8(IDS_AI_CHAT_QUESTION_SUMMARIZE_VIDEO_BULLETS);
+    question_part = l10n_util::GetStringUTF8(
+        IDS_AI_CHAT_CLAUDE_QUESTION_SUMMARIZE_VIDEO_BULLETS);
```

Comment:
This is submitted to all engine types (claude and llama), so it doesn't make sense to rename the string key to include the "claude" substring - were you attempting to accomplish something claude-specific here?

Reply:
Was not intentional. Reverted the change 👍